doc/lua_api.txt

```diff
@@ -4624,6 +4624,16 @@ Utilities
 * `minetest.mkdir(path)`: returns success.
     * Creates a directory specified by `path`, creating parent directories
       if they don't exist.
+* `minetest.rmdir(path, recursive)`: returns success.
+    * Removes a directory specified by `path`.
+    * If `recursive` is set to `true`, the directory is recursively removed.
+      Otherwise, the directory will only be removed if it is empty.
+* `minetest.cpdir(source, destination)`: returns success.
+    * Copies a directory specified by `path` to `destination`, `destination` is
```

Comment:
It is imprecise what "overwritten" means here.
Existing directories and files are not deleted *but* during copy files that already exist are overwritten.